pallets/fragments/src/lib.rs

```diff
@@ -304,6 +327,10 @@ pub mod pallet {
 	pub type Publishing<T: Config> =
 		StorageMap<_, Identity, Hash128, PublishingData<T::BlockNumber>>;
 
+	#[pallet::storage]
+	pub type Definition2SecondarySales<T: Config> =
```

Comment:
@sinkingsugar , **instead of it being** 
```
#[pallet::storage]
pub type Definition2SecondarySales<T: Config> =
StorageMap<_, Identity, Hash128, Vec<SecondarySaleData<T::AccountId, T::BlockNumber>>>;
```

**why can't we have it as:**

```
#[pallet::storage]
pub type Definition2SecondarySales<T: Config> = StorageNMap<
_,
(
	// Definition Hash
	storage::Key<Identity, Hash128>,
	// Edition ID
	storage::Key<Identity, Unit>,
	// Copy ID
	storage::Key<Identity, Unit>,
),
SecondarySaleData<T::AccountId, T::BlockNumber>,
>;
```

The storage used for to store an instance's `SecondarySaleData` would be the same. But by using `StorageNMap` it gives us the added benefit of querying and deleting an instance's `SecondarySaleData` in O(1) time.

Therefore, please let me know if we should use a `StorageNMap` instead. Thanks

Reply:
sounds good to me

Reply:
Okay. Thanks